spyder/app/mainwindow.py

```diff
@@ -2720,6 +2745,36 @@ def show_tour(self, index):
         self.tour.set_tour(index, frames, self)
         self.tour.start_tour()
 
+    # ---- Global File Switcher
+    def open_fileswitcher(self):
+        """Open file list management dialog box."""
+        if self.fileswitcher is not None and \
+          self.fileswitcher.is_visible:
+            self.fileswitcher.hide()
+            self.fileswitcher.is_visible = False
+            return
+        self.fileswitcher.setup()
+        self.fileswitcher.show()
+        self.fileswitcher.is_visible = True
+
+    def open_symbolfinder(self):
+        """Open symbol list management dialog box."""
+        self.open_fileswitcher()
+        self.fileswitcher.set_search_text('@')

```

Comment:
When opening the symbol switcher we should give focus to the Editor because it has no meaning for the other plugins, right?